packages/rich-text/src/store/test/selectors.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * Internal dependencies
+ */
+import { getFormatTypes, getFormatType } from '../selectors';
+
+describe( 'selectors', () => {
+	const defaultState = {
```

Comment:
It's a good practice to wrap state with `deepFreeze` to ensure that selectors never mutate state.